ldap/servers/slapd/back-ldbm/back-ldbm.h

```diff
@@ -653,14 +648,14 @@ struct ldbminfo
 #define LI_BDB_IMPL         0x20
 #define LI_LMDB_IMPL        0x40
 
-#define LI_DEFAULT_IMPL_FLAG  LI_BDB_IMPL /* the default is BDB for now */
+#define LI_DEFAULT_IMPL_FLAG  LI_BDB_IMPL /* the default is Bdbi_db_t for now */
 
 /* Structure used to hold stuff for the lifetime of an LDAP transaction */
 /* If we do clever stuff like LDAP transactions, we'll need a stack of TXN ID's */
 typedef struct back_txn back_txn;
 struct back_txn
 {
-    DB_TXN *back_txn_txn; /* Transaction ID for the database */
+    dbi_txn_t *back_txn_txn; /* Transaction ID for the database */
```

Comment:
We need to distinguish between read txn and write txn as these matter greatly to LMDB.

Reply:
I just added a section about read txn in the open points of the design document.
In short the read txn support is not something that I plan to do in phase 3.
The only thing is that we should have an architecture that could easily evolve to support it (and that is the case as we can add attribute to back_txn struct or within the  dbi_txn_t one.)